package.json

```diff
@@ -37,23 +38,22 @@
   "module": "dist/node/index.js",
   "types": "dist/node/index.d.ts",
   "scripts": {
-    "build": "node --experimental-loader esbuild-node-loader scripts/build.ts",
-    "test": "node --experimental-loader esbuild-node-loader --experimental-specifier-resolution=node ./node_modules/.bin/uvu tests",
+    "build": "node --experimental-loader esbuild-node-loader scripts/build.ts && tsc -p tsconfig.build.json",
+    "test": "vitest --globals",
     "prepublish": "npm run build"
   },
   "dependencies": {
-    "@remix-run/dev": "^1.0.6"
+    "@remix-run/dev": "^1.3.3"
   },
   "devDependencies": {
     "@types/react": "16.7.0",
-    "esbuild-node-loader": "^0.6.3",
+    "esbuild-node-loader": "^0.8.0",
```

Comment:
Bumped some dependencies when trying to debug why tests were failing. Ultimately wasn't needed, but still good to have.